pilot/pkg/kube/inject/testdata/inject/app_probe/ready_live.yaml.injected

```diff
@@ -67,13 +68,9 @@ spec:
         - --parentShutdownDuration
         - 1m0s
         - --discoveryAddress
-<<<<<<< HEAD
```

Comment:
is this file even getting tested.. shouldn't this have failed before?

Reply:
that is a good question, the merge git lines are there in master as well:
https://github.com/istio/istio/blob/master/pkg/kube/inject/testdata/inject/app_probe/ready_live.yaml.injected#L70-L76